paramak/parametric_components/center_column_circular.py

```diff
@@ -28,11 +28,11 @@ def __init__(
         workplane="XZ",
         rotation_angle=360,
         solid=None,
-        stp_filename="center_column.stp",
+        stp_filename="CenterColumnShieldCircular.stp",
         color=None,
         points=None,
         name="center_column",
-        material_tag="center_column_material",
+        material_tag="center_column_shield_material",
```

Comment:
```suggestion
        material_tag="center_column_shield_mat",
```

This one is the reason for changing material to mat as it is too long for the DAGMC limit